config/default/jenkins-infra.yaml

```diff
@@ -18,6 +18,11 @@ jenkins:
             system:
               domainCredentials:
                 - credentials:
+                  - gitHubApp:
+                      appID: "${GITHUB_APP_ID}"
+                      description: "GitHub App for infra.ci.jenkins.io"
+                      id: "github-app-infra"
+                      privateKey: "${GITHUB_APP_PRIVATE_KEY}"
                   - usernamePassword:
                       description: "GitHub access token for jenkinsadmin"
                       id: "github-access-token"
```

Comment:
I think we can safely remove this credential id

Reply:
Sounds right: https://github.com/search?q=org%3Ajenkins-infra+github-access-token&type=code

Reply:
As discussed together , we will deprecate the token on a subsequent PR. It sounds like there are no usage of the token outside the Jcasc's job-dsl conf, but we'll check this trough the Jenkins'UI (which references the build using a given token)